test-clients/src/main/java/com/hedera/services/bdd/suites/autorenew/GracePeriodRestrictionsSuite.java

```diff
@@ -124,7 +124,7 @@ private HapiApiSpec contractCallRestrictionsEnforced() {
                                                             asAddress(civilianAccountID.get()),
                                                             asAddress(detachedAccountID.get())
                                                         })
-                                                .hasKnownStatus(INVALID_SOLIDITY_ADDRESS)),
+                                                .hasKnownStatus(FAIL_INVALID)),
```

Comment:
Hmm.... this is failing on a transfer...

https://github.com/hashgraph/hedera-services/blob/3f44c6ec2853742971eb95cb5b6879b45c32e2a2/test-clients/src/main/resource/contract/solidity/DoubleSend.sol#L14

I think this is one case we do want to send an invalid solidity error,